wiki/Tournaments/oPHN/2024_taiko/en.md

```diff
@@ -0,0 +1,395 @@
+- The higher scores from each player for each map will be used for seeding. Players will be ordered by overall rank, which is computed by taking the average of their rankings on each map. In case of a tie, the sum of the top scores of the tied players are taken into account instead. Sharing of scores before the results showcase is discouraged.
+- The number of players that will advance to the next stage will depend on the total number of participants.
+
+#### Bracket Stage
+
+- Players are placed into a double-elimination bracket. They comprise of the Top 16 from the qualifiers.
+- Mappool Sizes and Best of Format for each round are given as follows:
+  - Round of 16 (Best of 9): 4 NM, 2 HD/HR/DT, 3 FM, 1 TB
+  - Quarterfinals and Semifinals (Best of 11): 5 NM, 2 HD/HR/DT, 3 FM, 1 TB
+  - Finals and Grand Finals (Best of 13): 6 NM, 2 HD/HR/DT, 3 FM, 1 TB
+
+### Match Rules
+
+- The multiplayer room will be created and invites sent to players 5-10 minutes prior to the match start time. The multiplayer rooms are named in the following format: `PHNT2024: (<Player 1>) vs. (<Player 2>)`
+- If a player is more than 10 minutes late to the for their match, the player inside the MP room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.
+- Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use DoubleTime if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.
+- Players are required to ban 1 map in the pool, excluding the tiebreaker. Banned charts cannot be picked during the match proper. The players shall use the `!roll` command in-game to determine the order in which charts are banned and picked. The order is as follows:
+  - Player with the higher roll: First Pick
+  - Player with the lower roll: First Ban
+  - The ban phase happens first, then the match proceeds to the picks.
+- Each player will alternate picks based on the `!roll` results until a winner is determined based on the Best-of format for the round. If both players are tied on match point (e.g. 4-4 score in a Best-of-9), the players will play the tiebreaker map of the pool to determine the winner.
+- Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random in behalf of the player.
+- Players are eligible to use a 2 minute tactical time out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
```

Comment:
```suggestion
- Each player will alternate picks based on the `!roll` results until a winner is determined based on the best-of format for the round. If both players are tied on match point (e.g. 4–4 score in a best-of-9), the players will play the tiebreaker map of the pool to determine the winner.
- Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random on behalf of the player.
- Players are eligible to use a 2-minute tactical time-out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
```